README.md

```diff
@@ -42,11 +42,15 @@ This stateless service sample demostrates writing a web frontend for your actor
 This stateless service demostrates a watch dog application which is monitoring the EchoServer application. Once you have depployed EchoServer and WatchDog application, you would see that if EchoServer application goes down or for some reason the applications web endpoint is not reachable, an error health is reported by WatchDog for the EchoServer application in the Service Fabric health subsystem. Apart from this, this application also shows inter service communication by forwarding the web requests to Echo Server application. You can view this by going to browser @ http://localhost:12345/getMessage to see the output coming from EchoServer. 
 
 ## Compiling the samples
-For compiling the samples use gradle. This should be configured as part of the Service Fabric Java SDK installation.
+For compilation the samples use gradle. This should be configured as part of the Service Fabric Java SDK installation.

```

Comment:
yeah. don't know why I changed it. probably too much reverse doc effect. updated it back to origin.